examples/storage_system.tf

```diff
@@ -1,50 +1,53 @@
 provider "oneview" {
-        ov_username = "<ov-username>"
-        ov_password = "<ov-password>"
-        ov_endpoint = "<ov-endpoint>"
-        ov_sslverify = false
-        ov_apiversion = <ov-apiversion>
-        ov_ifmatch = "*"
+  ov_username   = "<ov_username>"
+  ov_password   = "<ov_password>"
+  ov_endpoint   = "<ov_endpoint>"
+  ov_sslverify  = false
+  ov_apiversion = <ov_apiversion>
+  ov_ifmatch    = "*"
 }
 
+
 resource "oneview_storage_system" "ss_inst" {
-        hostname = "<hostname>"
-        username = "<user>"
-        password = "<password>"
-        family   = "<family>"
+        hostname = "172.18.11.12"
```

Comment:
Is there any reason for specifying values here?

Reply:
Removed the ips and credentials